components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java

```diff
@@ -321,7 +321,7 @@ protected void filterOptions(Map<String, Map<String, String>> authenticationOpti
 
                     if (FrameworkConstants.LOCAL_IDP_NAME.equals(idpName)) {
                         List<LocalAuthenticatorConfig> localAuthenticators = ApplicationAuthenticatorService
-                            .getInstance().getLocalAuthenticators();
+                            .getInstance().getAllSystemDefinedLocalAuthenticators();
```

Comment:
It should be possible to use both system defined and user defined authenticators in adaptive scripts. Doesn't this just allow system defined authenticators ?